java/generate-geodatabase/README.metadata.json

```diff
@@ -15,6 +15,12 @@
         "MapView",
         "ServiceFeatureTable"
     ],
+    "provision_from": [
+        "https://arcgisruntime.maps.arcgis.com/home/item.html?id=72e703cd01654e7796eb1ae75af1cb53"
+    ],
+    "provision_to": [
+        "/ArcGIS/samples/TileCache/SanFrancisco.tpk"
```

Comment:
not sure how much casing matters but, wondering if this should be `/Samples/` to match the resource string in the sample. Same applies for other instances of this in the PR

Reply:
I think it's ok, but thanks for checking!